src/d3a/models/strategy/predefined_load.py

```diff
@@ -126,3 +130,55 @@ def area_reconfigure_event(self, **kwargs):
         self._area_reconfigure_prices(**kwargs)
         if key_in_dict_and_not_none(kwargs, 'daily_load_profile'):
             self._event_activate_energy(kwargs['daily_load_profile'])
+
+
+class LoadForecastStrategy(DefinedLoadStrategy):
```

Comment:
I am fairly certain that we will never use this strategy without an external API. I would recommend to merge it with the LoadForecastExternalStrategy

Reply:
ok, makes sense!